src/components/image/index.js

```diff
@@ -1,23 +1,44 @@
 import React from "react"
 import PropTypes from "prop-types"
 import Img from "gatsby-image"
-import classNames from "classnames/bind"
+import classNames from "classnames"
 import dig from "object-dig"
+
 import styles from "./styles.module.scss"
 
 const Image = ({ alt, className, src, ...props }) => {
-  const classes = classNames(styles.cl_image, { [className]: className })
+  const defaultAltAttribute = image =>
+    image
+      .split("/")
+      .pop()
+      .split(".")
+      .slice(0, -1)
+      .join(".")
+
+  const classes = classNames(styles.image, { [className]: className })
 
   // ---------------------------------------- | Gastby Image
 
   if (dig(src, "childImageSharp", "fluid") || dig(src, "childImageSharp", "fixed")) {
-    return <Img className={classes} alt={alt} {...src.childImageSharp} />
+    let imageName = dig(src, "childImageSharp", "fluid", "src")
+
+    if (dig(src, "childImageSharp", "fixed")) {
+      imageName = dig(src, "childImageSharp", "fixed", "src")
+    }
```

Comment:
We're going to move away from `dig` and toward using `lodash` since we already have it available. #61 covers that, though, so this is fine.

I also feel like we could clean this up so we don't have an `if` statement for every object we're looking for.

Reply:
That said, I'm fine with this for now. It's refining and refactoring, so we can always come back to it.